internal/resume/templates/latex.go

```diff
@@ -38,6 +45,7 @@ var latexDocument = `
     \textheight=10in
     \pagestyle{empty}
     \raggedright
+    \usepackage{censor}
 
 %%%%%%%%%%%%%%%%%%%%%%% DEFINITIONS FOR RESUME %%%%%%%%%%%%%%%%%%%%%%%
 
```

Comment:
I think the `\StopCensoring` command should be added in the Latex template. Adding a `CensoringEnabled` boolean flag in the `Resume` struct in `internal/resume` seems like a good way to do it. Then in the template you could have `[[ if not .CensoringEnabled ]]\StopCensoring[[end -]]`. 

Doing it that way, it would be best to add censoring to the plaintext template as well, for consistency.

Reply:
I implemented this in the latest commit.